src/cli/repo-checkout.js

```diff
@@ -125,34 +125,47 @@ export default async function main(config, argv) {
     projects: {}
   };
 
-  await Promise.all(projects.map(async (project) => {
-    let projectStart = Date.now();
+  let archivesToExtract = await Promise.all(projects.map(async (project) => {
+    let downloadStart = Date.now();
+    stats.projects[project.name] = {duration: 0};
+
     let repoPath =
       fsPath.join(args.directory, '.repo', 'projects', `${project.path}.git`);
```

Comment:
I can't see if this is the target output directory or tar file location...
But if it's the target... and you're basically checking later that this already exists, because then you don't want to extract it....

Then why don't we just create this directory before we run tar... then I think parallel tar will work.
Don't they only operate on their own `XXXX.git/` folder? Don't know maybe it's too implementation specific dependency.

Guess we could leave such an optimization for later.